libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.h

```diff
@@ -41,6 +41,13 @@ typedef NS_ENUM(NSUInteger, SFAppType) {
     kSFAppTypeReactNative
 };
 
+typedef NS_ENUM(NSUInteger, SFAppFeature) {
```

Comment:
I don't think SalesforceSDKCore should know about the other libraries like SmartStore etc.
So instead appFeature could be a string.